### DISCUSSION TITLE
Mispell on sample output

### DISCUSSION THREAD

**Discussion by ramseshk** [ScrapeGraphAI/Scrapegraph-ai#340] [General]
On the README.md, the sample output is stated as: 

"{'recipes': [{'name': 'Sarde in Saòre'}, {'name': 'Bigoli in salsa'}, {'name': 'Seppie in umido'}, {'name': 'Moleche frite'}, {'name': 'Risotto alla pescatora'}, {'name': 'Broeto'}, {'name': 'Bibarasse in Cassopipa'}, {'name': 'Risi e bisi'}, {'name': 'Smegiassa Ciosota'}]

It's not "Saòre" but "Saor"
It's not "Moleche" but "Moeche" 
It's not "Bigoli" but "Bìgoi"

**Comment by itsperini** [comment]
Hey @ramseshk that's what the library found online and they correspond to the corrected names from Chioggia. You are referring to the Venician counterparts :)

**Comment by VinciGit00** [comment]
Bergamo

**Comment by VinciGit00** [comment]
ok please update to the new version
